tests/install_upgrade_operators/node_component/conftest.py

```diff
@@ -244,6 +246,7 @@ def alter_np_configuration(
         hco_namespace=hco_namespace,
         infra_placement=infra_placement,
         workloads_placement=workloads_placement,
+        exclude_deployments=[HYPERCONVERGED_CLUSTER_CLI_DOWNLOAD],
```

Comment:
if we  exclude from checks and it never becomes ready, there is a risk we never know? How we handle that

Reply:
Hi @geetikakay! 

IMO the cost to ensure this deployment is available is higher than the risk we are assuming. There are tests that guarantee that CLI can be downloaded: https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/install_upgrade_operators/console_cli_download/test_disconnected_virtctl.py

In the rare case that the pod cannot go up on a different node (which is what this test is doing), the functionality is not compromised as this deployment is not critical. Please take into account that this is a gating test.

I hope this clarifies. Let me know your thinking. Thanks!